### PR TITLE
gruppled-cursors, gruppled-lite-cursors: init at 1.0.0

### PR DESCRIPTION
**Pull request by nim65s**
###### Description of changes

Hi,

This is adding the gruppled cursors theme with 4 variants (black / white, lite or not)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by nim65s** [comment]
Result of `nixpkgs-review pr 242667` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>gruppled-black-cursors</li>
    <li>gruppled-black-lite-cursors</li>
    <li>gruppled-white-cursors</li>
    <li>gruppled-white-lite-cursors</li>
  </ul>
</details>

**Review by SuperSandro2000 — COMMENTED** [review]
We should set meta.description

**Comment by nim65s** [comment]
@SuperSandro2000 : yes, thanks ! done

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -28992,6 +28992,23 @@ with pkgs;
...
+      gruppled-white-cursors = callPackage ../data/icons/gruppled-cursors { theme = "gruppled_white"; };
+      gruppled-white-lite-cursors = callPackage ../data/icons/gruppled-lite-cursors {
+        theme = "gruppled_white_lite";
+      };
+    })
+    gruppled-black-cursors
+    gruppled-black-lite-cursors
+    gruppled-white-cursors
+    gruppled-white-lite-cursors
+    ;
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:29010]:
```suggestion
    gruppled-black-cursors = callPackage ../data/icons/gruppled-cursors {
      theme = "gruppled_black";
    };
    gruppled-black-lite-cursors = callPackage ../data/icons/gruppled-lite-cursors {
      theme = "gruppled_black_lite";
    };
    gruppled-white-cursors = callPackage ../data/icons/gruppled-cursors {
      theme = "gruppled_white";
    };
    gruppled-white-lite-cursors = callPackage ../data/icons/gruppled-lite-cursors {
      theme = "gruppled_white_lite";
    };
```
Isn't this just a more complicated form of this?

Comment (nim65s) [reply pkgs/top-level/all-packages.nix:29010]: This is what I used initially, but the by-name ci was not happy with it.
Therefore I asked on https://discourse.nixos.org/t/rfc140-what-about-package-with-multiple-names/41846

Comment (SuperSandro2000) [reply pkgs/top-level/all-packages.nix:29010]: I don't like it either...

**Pull request merged**